src/index.html

```diff
@@ -2,13 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Title</title>
+    <title>Tree from Object</title>
     <link
       rel="stylesheet"
       href="./styles/main.scss"
```

Comment:
The `href` attribute is linking to a SCSS file (`main.scss`). Browsers cannot directly interpret SCSS files. You should compile the SCSS file to a CSS file and link to the resulting CSS file instead.